Improve Documentation (#339)
hivemq/mqtt-cli: docs/_includes/options/subscribe-options.md

| Option | Long Version     | Explanation                                                                                                                                                                                                                           | Default |
|--------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
| `-t`   | `--topic`        | The MQTT topic the client will subscribe to.                                                                                                                                                                                          |         |
| `-q`   | `--qos`          | Define the quality of service level. If only one QoS is specified it will be used for all topics.<br> You can define a specific QoS level for every topic. The corresponding QoS levels will be matched in order to the given topics. | `0`     |

> | `-of`  | `--outputToFile` | If a file is given print the received publishes to the specified output file. If the file is not present it will be created.                                                                                                          |         |

Reviewer: ```suggestion
| `-of`  | `--outputToFile` | Append the received publish messages to a file. Creates the file if it does not exist.                                                                                                           |         |
```
